sources/cfsignal/src/cloudformation.rs

```diff
@@ -0,0 +1,65 @@
+use crate::error::{self, Result};
+use imdsclient::ImdsClient;
+use log::info;
+use rusoto_cloudformation::{
+    CloudFormation, CloudFormationClient, SignalResourceInput,
+};
+use rusoto_core::Region;
+use snafu::OptionExt;
+use snafu::ResultExt;
```

Comment:
```suggestion
use snafu::{OptionExt, ResultExt};
```

One of these commands should catch or fix stuff like this:
```
cargo check -p cfsignal
cargo clippy -p cfsignal -- --no-deps
cargo fmt -p cfsignal
```

Reply:
just for info, none of the above catch that issue.
(but using `clippy` i found other suggestions, for other part of code, to adopt)